src/lando/api/tests/test_landings.py

```diff
@@ -519,27 +583,27 @@ def test_merge_conflict(
         ), f"Empty or missing reject content for failed path {fp}"
 
 
+@pytest.mark.parametrize(
+    "repo_type",
+    [
+        # Bug 1940876
+        # SCM_TYPE_GIT,
+        SCM_TYPE_HG,
+    ],
+)
 @pytest.mark.django_db
 def test_failed_landing_job_notification(
-    hg_server,
-    hg_clone,
+    repo_mc,
     treestatusdouble,
     monkeypatch,
     create_patch_revision,
+    repo_type: str,
 ):
     """Ensure that a failed landings triggers a user notification."""
-    treestatusdouble.open_tree("mozilla-central")
-    repo = Repo.objects.create(
-        scm_type=SCM_TYPE_HG,
-        name="mozilla-central",
-        required_permission=SCM_LEVEL_3,
-        push_path=hg_server,
-        pull_path=hg_server,
-        approval_required=True,
-        autoformat_enabled=False,
-        system_path=hg_clone.strpath,
-    )
-
+    repo = repo_mc(repo_type)
+    repo.approval_required = True
+    repo.autoformat_enabled = False
```

Comment:
Same comment as above -- ideally these are just passed as overrides to the fixture.